app/uk/gov/hmrc/bindingtariffclassification/model/Decision.scala

```diff
@@ -33,6 +35,5 @@ case class Decision
 
 case class Appeal
 (
-  reviewStatus: String,
-  reviewResult: String
```

Comment:
The only thing that can currently add appeal info is the acceptance tests. So there shouldn't be any migration involved